### DISCUSSION TITLE
Content Review Process and Edits

### DISCUSSION THREAD

**Discussion by zackarychapple** [forem/forem#18918] [Feature Requests]
We are starting to more heavily use the Organization functionality for our team and are putting out more content. There are a few quality of life improvements that would be very nice to have. 

The ability to be able to edit / suggest changes on a preview link.  It would be great if when someone gives us their preview link if we can review that content directly and suggest edits / comments / insert assets directly for them to approve or decline (similar to how google docs works). 

The second request is for our team to be able to submit articles that are ready for publish, and allow us to be able to hit the publish on their behalf when the article is approved.

**Comment by antweiss** [comment]
Yes! This! So much needed! In the last year I've reviewed a couple dozens posts and it's always such a pain. Sorry to say but it makes me miss Medium... :(
